content/en/community/roadmap.md

```diff
@@ -9,28 +9,26 @@ weight: 100
 The Jenkins X roadmap will be updated approximately once a month. 
 Please join our [office hours](https://jenkins-x.io/community/#office-hours) and [Slack channels](/community/#slack) to ask questions and drive input on the future of the project!
 
-Please note that the [Current Work](/community/roadmap/#current-work) is being done in Jenkins X 2 and that most of the listed [Near Term](/community/roadmap/#near-term-work) and [Future Work](/community/roadmap/#future-work) are proposals that are being discussed in the [Jenkins X enhancements repo](https://github.com/jenkins-x/enhancements) and will likely form the basis of Jenkins X 3. Some of the proposed enhancements will land in Jenkins X 2 as well, but the focus for future work is on Jenkins X 3. Read more about [Jenkins X 3 alpha](https://jenkins-x.io/v3/)
-
+Please note that the [Current Work](/community/roadmap/#current-work) is being done in Jenkins X 3 as Jenkins X 2 is no longer being maintained.
 ## Current Work
 
-- [Bring your own vault](https://jenkins-x.io/docs/install-setup/installing/boot/secrets/#external)
-- Granularized permissions
-- OpenShift support
-- [Custom buildpacks](https://jenkins-x.io/docs/create-project/build-packs/#using-custom-build-packs)
-- Terraform cloud orchestration
-- Kubernetes 1.16 compatibility
-- Bring your own container registry
+- Improving our documentation and depreceating any documentation that is now defunct
+- Kubernetes 1.22 support
+- Making our Terraform and GitOps modules inline with one another
+- Introducting security scanning and container signing
+- Build our community and get more stakeholders and users at the table
```

Comment:
We should also add linting and upgrading to go1.17.
So current work is what we are working on now, right? We should update the board to reflect this. I will move a few tickets.